src/ResourceManager/PowerBIDedicated/Commands.PowerBIDedicated/Commands/TestAzureRmPowerBIEmbeddedCapacity.cs

```diff
@@ -0,0 +1,41 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.PowerBIDedicated.Models;
+using Microsoft.Azure.Management.PowerBIDedicated;
+using System.Management.Automation;
+using Microsoft.Azure.Management.PowerBIDedicated.Models;
+
+namespace Microsoft.Azure.Commands.PowerBIDedicated
+{
+    [Cmdlet(VerbsDiagnostic.Test, "AzureRmPowerBIEmbeddedCapacity"), OutputType(typeof(bool))]
+    public class TestAzurePowerBIEmbeddedCapacity : PowerBIDedicatedCmdletBase
+    {
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
+            HelpMessage = "Name of a specific capacity.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = false,
```

Comment:
@selasagi I know I asked in the review, but why isn't `ResourceGroupName` required?

Reply:
Because the Name is unique so it is enugh to the Test and Remove cmdlets